src/tickit/adapters/epicsadapter/adapter.py

```diff
@@ -73,14 +73,15 @@ def on_db_load(self) -> None:
 
     def load_records_without_DTYP_fields(self):
         """Load records from database file without DTYP fields."""
-        with open(self.db_file, "rb") as inp:
-            with NamedTemporaryFile(suffix=".db", delete=False) as out:
-                for line in inp.readlines():
-                    if not re.match(rb"\s*field\s*\(\s*DTYP", line):
-                        out.write(line)
-
-        softioc.dbLoadDatabase(out.name, substitutions=f"device={self.ioc_name}")
-        os.unlink(out.name)
+        if self.db_file:
```

Comment:
Does this need rebasing on top of @joeshannon's change?

Reply:
#134? It's already merged

Reply:
Oh yeah, that's not in there, also worrying that got past mypy...